FreeRTOS_TCP_WIN.c

```diff
@@ -217,7 +218,7 @@
 
 /**< TCP segment pool. */
     #if ( ipconfigUSE_TCP_WIN == 1 )
-        static TCPSegment_t * xTCPSegments = NULL;
+        _static TCPSegment_t * xTCPSegments = NULL;
```

Comment:
Why not let the script remove the `static` qualifier and then use it?
You can find this file in the Aneexed_TCP folder with all `static` qualifiers removed.